integrationtest/spring/application/pom.xml

```diff
@@ -0,0 +1,88 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.hibernate.search</groupId>
+        <artifactId>hibernate-search-integrationtest-spring-repackaged</artifactId>
+        <version>7.0.0-SNAPSHOT</version>
+        <relativePath>..</relativePath>
+    </parent>
+    <artifactId>hibernate-search-integrationtest-spring-repackaged-application</artifactId>
+
+    <name>Hibernate Search ITs - Spring Repackaged JAR Application</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.hibernate.search</groupId>
+            <artifactId>hibernate-search-integrationtest-spring-repackaged-model</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Do you need dependencies to test this? I would have expected the uberjar to put the application classes in a nested jar as well... ?

If you do need dependencies, that's actually good news as this means far fewer applications are affected than I originally thought.

Reply:
Yup. That's a +. Spring has that boot-inf folder in the jar, and then there is a classes folder with classes from the "current module" and a lib folder with all the jars, so yeah, it should only affect those who put their entities etc. in some other jar.